docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -44,11 +44,11 @@ In sbt:
 
 @[cassandra-clean-on-start](code/build-cassandra-opts.sbt)
 
-# Keyspace
+## Keyspace _(deprecated)_
```

Comment:
I failed to notice this was the `devmode` page.
In any case, a clear statement on why each service needs a separate keyspace is required. After re-reading other pages I see it's all covered.


PS: what I tried to say in my previous comments was that some info had to be kept: "A keyspace in Cassandra is a namespace that defines data replication on nodes. Each service should use a unique keyspace name so that the tables of different services do not conflict with each other."
That info is now in other pages already.